tsconfig.build.json

```diff
@@ -1,4 +1,4 @@
 {
   "extends": "./tsconfig.json",
-  "exclude": ["node_modules", "test", "dist", "**/*spec.ts"]
+  "exclude": ["node_modules", "test", "dist", "**/*spec.ts", "**/*mock.ts"]
```

Comment:
Why do you need to exclude mocks?

Reply:
It's more that I don't have to include them. Neither tests nor api will use the compiled mocks, so why compile them?